packages/cdktf-cli/templates/python/Pipfile

```diff
@@ -4,4 +4,4 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [requires]
-python_version = "3.7"
\ No newline at end of file
+python_version = "3"
```

Comment:
I don't think that'll work - see https://github.com/awslabs/cdk8s/blob/5759d0d927268bb4e9cb9ba01d13053b44834fed/docs/getting-started/python.md#prerequisites

Reply:
We might want to remove entirely then or switch away from pipenv since something like ">=3.7" isn't supported.

Reply:
Hm, when looking at the [jsii build config](https://github.com/aws/jsii/blob/6ea8daa8a022c5bda5fd1854d49292e65caac2be/.github/workflows/main.yml) it looks like at least 3.6 - 3.8 should work. And according to [this](https://www.python.org/downloads/) 3.5 won't be supported soon. 

![Screenshot 2020-07-31 at 16 15 14](https://user-images.githubusercontent.com/136789/89043685-28467180-d349-11ea-82d8-e2108f78cdc3.png)

@eladb are there docs which specify Python version support for `jsii`?

Reply:
Well, let's just go with the major version then. Seems to work fine.